types/mathjs/index.d.ts

```diff
@@ -2249,4 +2250,11 @@ declare namespace math { // tslint:disable-line strict-export-declare-modifiers
 		valueOf(): any;
 		toString(): string;
 	}
+
+	interface MathJsJson {
+		/**
+		 * Returns reviver function that can be used as reviver in JSON.parse function.
+		 */
+		reviver(): (key: any, value: any) => any;
```

Comment:
I was tempted to see if we could do any better than `any`, `any`, `any`.

If we wanted to be a little stronger we could at least say `key` should be `string`, but I'm not sure that we can improve `value` or the return.